src/components/Bar.svelte

```diff
@@ -0,0 +1,97 @@
+<button class='coz-bar-burger' on:click='toggleDrawer()'>
+  <svg class='coz-nav-icon'><use xlink:href='{{burgerIcon}}' /></svg>
+</button>
+
+<h1 class='coz-bar-title'>
+  <img class='coz-bar-hide-sm' src='{{ iconPath }}' width='32' />
+  <span class='coz-bar-hide-sm'>cozy </span><strong>{{ appName }}</strong>
+</h1>
+
+<hr class='coz-sep-flex' />
+
+<Navigation sections='{{ sections }}' />
+
+<Drawer ref:drawer groups='{{sections[0].items}}' />
+
+<script>
+  import Navigation from './Navigation'
+  import Drawer from './Drawer'
+
+  import MENU_CONFIG from '../config/menu'
+
+  export default {
+    data () {
+      return {
+        burgerIcon: require('../assets/icons/icon-hamburger.svg'),
+        sections: [{
+          label: 'settings',
+          icon: require('../assets/icons/icon-cog.svg'),
+          items: MENU_CONFIG.settings
+        }]
+      }
+    },
+
+    components: {
+      Navigation,
+      Drawer
+    },
+
+    methods: {
+      toggleDrawer() {
+        this.refs.drawer.set({folded: false})
+      }
+    }
+  }
+</script>
+
+<style>
+  .coz-bar-title {
+    display: flex;
+    margin: 0;
+    align-items: center;
+    font-size: 1.5em;
+    font-weight: normal;
+    text-transform: lowercase;
+    color: var(--charcoal-grey);
+  }
+
+  @media (max-width: 30em) {
+    .coz-bar-title {
+      font-size: 1.25em
+    }
+  }
+
+  .coz-bar-title img {
+    margin-right: .45em;
+  }
+
+  .coz-bar-title strong {
+    margin-left: .25em;
+    font-weight: bold;
+  }
+
+  @media (max-width: 30em) {
```

Comment:
I'me used to declare media-queries that directly follow the component they modify, as it seems to be more simple to debug / maintain. Postcss will they pack all MQ at the end when the final CSS is built. Does this sounds good, or do you prefer we adopt another convention?